public/app/features/alerting/unified/components/rule-viewer/tabs/Details.tsx

```diff
@@ -92,7 +95,7 @@ const Details = ({ rule }: DetailsProps) => {
           )}
         </MetaText>
         <MetaText direction="column">
-          {pendingPeriod && (
+          {!isRecordingRulerRule(rule.rulerRule) && !isGrafanaRecordingRule(rule.rulerRule) && pendingPeriod && (
             <>
```

Comment:
Can we move this logic to the `usePendingPeriod` hook? That way we don't need to duplicate in many places

Reply:
good idea 👍